### ISSUE TITLE
address private key

### ISSUE THREAD

**Issue by 999ethpo** [iceland2k14/ETH_Hunt#4]
hi, I don't understand this is the same v3 eth script searching for a key by address from the eth_address file? and where does he save the key after he has found it? there is no such thing in the script code so that it can be saved to a file

**Comment by iceland2k14** [comment]
Yes it is searching and checking the match with address in the file.
The Key is displayed on Screen if match is found. Not saved anywhere currently.

**Comment by rammos92** [comment]
Found and Stay on the Top!
